src/main/java/io/fabric8/maven/docker/access/log/LogRequestor.java

```diff
@@ -224,6 +227,7 @@ public void finish() {
                 }
             }
         }
+        callback.close();
```

Comment:
finish() is called in case of an exception but the `callback.close()` is also included in the finally clauses of the top level methods. This leads to closing a callback() twice which is probably not intended.